adaptive_components/example/lib/adaptive_column_example.dart

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:adaptive_components/adaptive_components.dart';
+
+void main() {
+  runApp(AdaptiveColumnsExample());
+}
+
+class AdaptiveColumnsExample extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        body: AdaptiveColumn(
+          children: [
+            AdaptiveContainer(
+              adaptiveConstraints: AdaptiveConstraints.xsmall(),
+              adaptiveColumn: 2,
+              color: Colors.red,
+              child: Text('This is an extra small window'),
+            ),
+            AdaptiveContainer(
+              adaptiveConstraints: AdaptiveConstraints.small(),
```

Comment:
These fields should probably drop the `adaptive` prefix, since they exist within an AdaptiveContainer

Reply:
Should I change the name of the class as well. I thought having adaptive as a prefix to this package would help users in terms of autocomplete.

Reply:
I think this is what was meant. Right @clocksmith ?
```suggestion
              constraints: AdaptiveConstraints.small(),
```